### ISSUE TITLE
Is development on this theme dead?

### ISSUE THREAD

**Issue by Dazeh** [320press/wordpress-bootstrap#235]
The reason i ask is because the site http://320press.com/ is gone and there hasnt been any updates to the theme in months.

**Comment by eduardoarandah** [comment]
I am worried about the same thing. 

I proposed a bugfix but wasnt merged. 

If so... How can we Take over it? Is there a procedure?

**Comment by marcbrigham** [comment]
It would be great if the community took this over. You will need to fork this repo and make changes to it.

**Comment by eduardoarandah** [comment]
Is there a way to make a fork and allow  multiple contributors? 

I know how to fork but don't have deep understanding of how everything works

**Comment by marcbrigham** [comment]
This link should give you some direction. 
https://help.github.com/articles/adding-collaborators-to-a-personal-repository/

**Comment by eduardoarandah** [comment]
If nobody answers what do you think is the best way to go? 

1) Make a Fork and try to tell everybody?
2) Create new repo from scratch

**Comment by dsghi** [comment]
I'd be willing to contribute as well, I have a client that has a third party custom theme derived from this and I have some updates that need to be implemented. I just don't have time/want to manage the repo :p

**Comment by touchst** [comment]
I would be willing to help as well I use a non preprocessor version of this for all my projects that I forked from this version, I have already integrated quite a few other things that could act as plugins.

**Comment by eduardoarandah** [comment]
Great! 

Give me a week to learn how to do this well 

What do you think is best?
  fork it or create a new repo "wordpress-bootstrap 2"

**Comment by marcbrigham** [comment]
Imo I would fork it so that it doesn't look like you just stole the code. One thing I have always hated was the name of the repo. But that's just m being picky. This might be a good time to change the name of others feel the same way as me.

**Comment by eduardoarandah** [comment]
Well, the name gives a 100% coincidence when you google "wordpress bootstrap"

Thats how I Got here. 

What do you suggest?

**Comment by marcbrigham** [comment]
Yeah I suppose your right. It's #6 on google. I'm fine with the name.

**Comment by thestepafter** [comment]
I can offer free web hosting if you would like information about this separate from Github.

**Comment by jcchavezs** [comment]
I would like to contribute on this. I was about making a pull request but I am afraid none can process it?

**Comment by Dazeh** [comment]
I would be more than happy to help you guys out with this. We havent heard a thing from the original developers so i guess the best thing to do is for us to take control of it. 

Just out of curiosity has anyone tried to contact the original developers?

**Comment by eduardoarandah** [comment]
I guess that's the first thing to do. Lets find them on fb tw etc 

Can the owner grant us permissions for merging to master?

**Comment by mhautamaki** [comment]
So did you guys decide on what to do with this? I'm thinking about just doing it.. if you guys are going to take days to decide. Any luck contacting them?

**Comment by marcbrigham** [comment]
I don’t have the time to contact them but since the developers made this repo they should be getting these comments in an email at the very least and should have a good idea whats going on.

**Comment by mhautamaki** [comment]
Can we bump this thing?

**Comment by eduardoarandah** [comment]
I contacted one of the contributors, I might get push access. 
Anyone else interested ?
Contact Noah Bass <noah@noahbass.com>

**Comment by jcchavezs** [comment]
Hi,

What is the status of the project? Can I contribute with it?

**Comment by eduardoarandah** [comment]
Looks like developers abandoned it. 

I gave up in favor to roots/sage which is pretty much the same so your knowlege won't be lost:
https://github.com/roots/sage

Only difference is that you have to 
npm install
bower install 
gulp 
before using it. 

After that, just rewrite variables in bower_components/bootstrap/variables.less to the main.less file
and "gulp watch"

What I like about that theme is that is mantained by a company so it wont be abandoned soon

**Comment by RussellTaylor83** [comment]
I have just bumped into this, having found the 320press.com website now dead.

I have created a fork here, https://github.com/RussellTaylor83/wordpress-bootstrap, if anyone wants to contribute their changes to it then please do and I'll review/merge and open it up to others to manage.

I would like to change the homepage to a single sidebar arrangement, so if I get time I'll make that my first update.

Thanks, it's a shame as the 320press stuff worked really well.

**Comment by eduardoarandah** [comment]
Hi everybody. I recommend moving to this project instead https://roots.io/sage/

Its very active, modern and useful. 

It also uses bootstrap and other great tecnologies
